software/base/src/main/java/org/apache/brooklyn/entity/software/base/AbstractSoftwareProcessWinRmDriver.java

```diff
@@ -39,8 +44,11 @@
 
 import com.google.api.client.util.Strings;
 import com.google.common.collect.ImmutableList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public abstract class AbstractSoftwareProcessWinRmDriver extends AbstractSoftwareProcessDriver {
+    private static final Logger LOG = LoggerFactory.getLogger(AbstractSoftwareProcessDriver.class);
```

Comment:
Please change the logger to `LoggerFactory.getLogger(AbstractSoftwareProcessWinRmDriver.class)`